.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: cpp
+compiler:
+  - clang
+sudo: false
+cache: ccache
+matrix:
+  include:
+    - os: linux
+      node_js: "latest"
+      script:
+        - NODE=$(which node) make lint-ci
+    - os: linux
```

Comment:
@thefourtheye I’m not sure I know exactly what you’re asking, but we defined `node_js` and `script` separately for this job because this is only for the linter